app/js/services/cp_analytics_api_client.js

```diff
@@ -3,3 +3,15 @@ import { get, post, parseJSON } from './auth'
 
 export const getTableauToken = () => get(`${CP_ANALYTICS_API}tableau/token`).then(parseJSON)
 export const postStats = (type, key, value) => post(`${CP_ANALYTICS_API}stats`, { type, key, value })
+export const loginToSentisum = () => {
+  let newTab = window.open('', '', '')
+
+  return get(`${CP_ANALYTICS_API}sentisum/login`)
+    .then(parseJSON)
+    .then(json => { newTab.location = json.url })
+    .catch((e) => {
```

Comment:
I'm not sure I understand the behaviour in case of an error.
Is it something like this?
  * We open the tab
  * we do the GET request and it fails
  * We close the tab and we show the alert.

If it is so, maybe it is better to open the tab *only if* the GET succeeds?

Reply:
@costis you understand it well. But there is a reason why we create new tab before the request is made. The reason is that opening new tab, has to be done immediately on user action "click". If you will do it on ajax callback, the browser will block popup. https://stackoverflow.com/questions/4602964/how-do-i-prevent-google-chrome-from-blocking-my-popup